src/sentry/similarity/__init__.py

```diff
@@ -121,7 +121,7 @@ def _make_index_backend(cluster=None):
         'message:message:character-shingles': MessageFeature(
             lambda message: text_shingle(
                 5,
-                message.message,
+                message.formatted,
```

Comment:
"generally interfaces shouldn't change" / "just use the attribute and move on"

Reply:
Also worth noting for future me that this doesn't really work the same way (interpolated messages will have higher cardinality going into the feature set, so probably cause more Redis issues) but I don't care enough to drag this out